### PR TITLE
DOCS-8916 Add Using LLM Observability and APM Guide

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Adds a guide with screenshots and videos on navigating to/from spans in LLM Observability and APM so users can achieve end-to-end visibility on their LLM applications.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/llm_observability/guide/llm_observability_and_apm.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  tag: "Documentation" 
+  text: "Learn about LLM Observability Spans"
+- link: "/glossary/#span/" 
+  tag: "Documentation" 
+  text: "Learn about APM Spans"
+---
+
+## Overview
+
+The LLM Observability SDK is built on APM's `dd-tracer`. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]. By instrumenting your LLM-specific operations with LLM Observability and your broader application with APM, LLM Observability and APM [spans][6] are linked in Datadog.
```
Comment (hestonhoffman) [inline content/en/llm_observability/guide/llm_observability_and_apm.md:15]:
```suggestion
The LLM Observability SDK is built on APM's `dd-tracer`. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]. You can link LLM Observability and APM [spans][6] in Datadog by instrumenting your LLM-specific operations with LLM Observability and your broader application with APM.
```
The last sentence tripped me up a bit. Rephrasing also gets rid of the passive.

File: content/en/llm_observability/guide/llm_observability_and_apm.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+This integration allows you to accomplish the following:
+
+* Understand end-to-end visibility: Explore upstream and downstream requests of your LLM applications within the context of your entire application.
+* Dive deeper from APM to LLM Observability to investigate whether or not an issue with your application is specific to LLM-specific applications, such as a call to OpenAI.
```
Comment (hestonhoffman) [inline content/en/llm_observability/guide/llm_observability_and_apm.md:20]:
```suggestion
* From APM, dive deeper into LLM Observability: Investigate whether or not an issue with your application is specific to LLM-specific applications, such as a call to OpenAI.
```

File: content/en/llm_observability/guide/llm_observability_and_apm.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Setup
+
+If you are using the [LLM Observability SDK for Python][1] along with APM's [`dd-tracer`][2], you can navigate between spans in Datadog APM and LLM Observability without any additional setup.
```
Comment (hestonhoffman) [inline content/en/llm_observability/guide/llm_observability_and_apm.md:24]:
```suggestion
If you are using the [LLM Observability SDK for Python][1] along with APM's [`dd-tracer`][2], you can navigate between spans in Datadog APM and LLM Observability without additional setup.
```

**Review by hestonhoffman — APPROVED** [review]
LGTM. Left a couple small suggestions

File: content/en/llm_observability/guide/llm_observability_and_apm.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  tag: "Documentation" 
+  text: "Learn about LLM Observability Spans"
+- link: "/glossary/#span/" 
+  tag: "Documentation" 
+  text: "Learn about APM Spans"
+---
+
+## Overview
+
+The LLM Observability SDK is built on APM's `dd-tracer`. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]. You can link LLM Observability and APM [spans][6] in Datadog by instrumenting your LLM-specific operations with LLM Observability and your broader application with APM.
```
Comment (barieom) [inline content/en/llm_observability/guide/llm_observability_and_apm.md:15]:
```suggestion
This guide explains how you can use both LLM Observability and APM to link LLM Observability and APM [spans][6] in Datadog. 

By instrumenting your LLM-specific operations with LLM Observability and your broader application with APM, you can accomplish the following:

```

File: content/en/llm_observability/guide/llm_observability_and_apm.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+This integration allows you to accomplish the following:
```
Comment (barieom) [inline content/en/llm_observability/guide/llm_observability_and_apm.md:17]:
```suggestion

```

File: content/en/llm_observability/guide/llm_observability_and_apm.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+* Understand end-to-end visibility: Explore upstream and downstream requests of your LLM applications within the context of your entire application.
+* From APM, dive deeper into LLM Observability: Investigate whether or not an issue with your application is specific to LLM-specific applications, such as a call to OpenAI.
+
+## Setup
+
```
Comment (barieom) [inline content/en/llm_observability/guide/llm_observability_and_apm.md:23]: Add
> The LLM Observability SDK is built on APM's `dd-tracer`. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7].

**Review by barieom — COMMENTED** [review]
Left a few comments, lgtm otherwise, thank you!

File: content/en/llm_observability/guide/llm_observability_and_apm.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+By instrumenting your LLM-specific operations with LLM Observability and your broader application with APM, you can accomplish the following:
+
+
+
+* Understand end-to-end visibility: Explore upstream and downstream requests of your LLM applications within the context of your entire application.
+* From APM, dive deeper into LLM Observability: Investigate whether or not an issue with your application is specific to LLM-specific applications, such as a call to OpenAI.
+
+## Setup
+
+The LLM Observability SDK is built on APM's dd-tracer. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]
```
Comment (rtrieu) [inline content/en/llm_observability/guide/llm_observability_and_apm.md:26]:
```suggestion
The LLM Observability SDK is built on APM's dd-tracer. This allows you to use LLM Observability with [Application Performance Monitoring (APM)][7]

```

**Pull request merged**